bus/bus.go

```diff
@@ -1572,15 +1601,17 @@ func (b *bus) settingKeyHandlerGET(jc jape.Context) {
 		jc.Error(err, http.StatusInternalServerError)
 		return
 	}
-
 	var resp interface{}
 	err = json.Unmarshal([]byte(setting), &resp)
 	if err != nil {
 		jc.Error(fmt.Errorf("couldn't unmarshal the setting, error: %v", err), http.StatusInternalServerError)
 		return
 	}
-
-	jc.Encode(resp)
+	if key == "s3authentication" {
```

Comment:
why only return the `s3authentication` setting as a prometheus metric?

Reply:
the other data points are already being collected elsewhere for the other settings so it wasn't needed.